packages/components/Item/Item.js

```diff
@@ -53,6 +56,7 @@ const ItemComponent = ({
   const [isVisibleDragZone, setVisibleDragZone] = useState(false);
   const itemRef = useRef(null);
   const notification = useNotification();
+  const isDecryptionProcess = useSelector(isDecryptionProgressSelector);
```

Comment:
I think better use isDecryptionProcess or isDecryptionProgress (to make variable consistent with selector)

Reply:
Fixed